packages/format-library/src/link/inline.js

```diff
@@ -38,45 +31,6 @@ function isShowingInput( props, state ) {
 	return props.addingLink || state.editLink;
 }
 
-const LinkEditor = ( { value, onChangeInputValue, onKeyDown, submitLink, autocompleteRef } ) => (
-	// Disable reason: KeyPress must be suppressed so the block doesn't hide the toolbar
```

Comment:
While it's technically fair to say the comment was no longer necessary because the changes remove the need to disable the ESLint rule, there was some value in having an explanation here previously for _why_ we stop propagation. Considering the purpose of a specific `Event#stopPropagation` is almost never easily understandable at a glance, we should strive to retain as much information for their existence as possible.

Coincidentally, I see it as similar to why we have these "Disable reason:" comment, in that it's similarly non-obvious to a future maintainer why we disable an ESLint rule without a comment explaining the rationale (and, to an arguably greater effect, forcing ourselves to explain it might actually deter us from going through with hastily introducing an escape hatch).

For context: I arrived here after trying to determine whether I could remove the `stopKeyPropagation` in a refactor, because I can't understand based on the current code why it needs to exist.